service/src/main/resources/db/changelog/changesets/20240228_add_fence_account_key_and_dist_lock_table.yaml

```diff
@@ -0,0 +1,45 @@
+databaseChangeLog:
+  - changeSet:
+      id: "add_fence_account_key_and_dist_lock_table"
+      author: sehsan
+      changes:
+        - createTable:
+            tableName: fence_account_key
```

Comment:
I think we also need a foreign key to the user's `linked_account` record, so that we can retrieve a user's fence account key by user_id and provider (similar to [ga4gh_passport](https://github.com/DataBiosphere/terra-external-credentials-manager/blob/dev/service/src/main/resources/db/changelog/changesets/20210521_initial_schema.yaml#L56)). Does that make sense @tlangs?

Reply:
That does make sense!